source/release-notes/1.7.txt

```diff
@@ -15,6 +15,52 @@ Release Notes for mongosync 1.7
 Patch Releases
 --------------
 
+.. _1.7.2-c2c-release-notes:
+
+1.7.2 Release
+~~~~~~~~~~~~~
+
+**February 21, 2024**
+
+Issues Fixed:
+
+- Fixed a bug causing mongosync to error out due to index
+  inconsistencies in background keys between shards, which the
+  server ignores.
+- Fixed a bug where mongosync could miss change events if stopped
+  or paused then resumed during the collection copy phase.  
+
+  This can happen when the destination cluster runs MongoDB 4.x
+  and the migration includes collections with mixed ``_id``
+  types or when the destination is MongoDB 6.0+ and the
+  migration includes capped collections.
+
+- Fixed a bug where indexes could be created on the destination
+  cluster during migration even when dropped on the source
+  cluster.
+
+- Fixed a bug where change events could be applied to the wrong
+  collection during concurrent collection renames.
+
+Other Changes:
+
+- Disables the Oplog Rollover Resilience machanism. 
+
+- Live upgrade to mongosync 1.7.2 is not allowed.
+
+- Reduces latency of the mongosync API.
+
+Limitations:
+
+- mongosync must read from the source cluster using the 
+  :readmode:`primary` read preference.
+
+- The balancer must be stopped on a sharded source cluster until
+  until mongosync finishes the collection copy phase.
+
+- The :dbcommand:`moveChunk` and :dbcommand:`moveRange` commands
+  must not be run on the source cluster.
+
```

Comment:
ah I see that the phrasing for the bullets on the other pages is from the release notes - would we be able to make these active voice or is there a preference for not touching the language generated in the changeling?